hl-MGMT-operations/java/src/main/java/com/example/healthlake/HealthlakeApplication.java

```diff
@@ -0,0 +1,51 @@
+package com.example.healthlake;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.healthlake.HealthLakeClient;
+import software.amazon.awssdk.services.healthlake.model.DescribeFhirDatastoreRequest;
+import software.amazon.awssdk.services.healthlake.model.DescribeFhirDatastoreResponse;
+import software.amazon.awssdk.services.healthlake.model.ListFhirDatastoresRequest;
+import software.amazon.awssdk.services.healthlake.model.ListFhirDatastoresResponse;
+
+@SpringBootApplication
+public class HealthlakeApplication implements CommandLineRunner {
+
+    private static Logger LOG = LoggerFactory.getLogger(HealthlakeApplication.class);
+
+    public static void main(String[] args) {
+        LOG.info("Running your spring boot console application");
+        SpringApplication.run(HealthlakeApplication.class, args);
+    }
+
+    @Override
+    public void run(String... args) {
+
+        Region region = Region.US_EAST_1;
```

Comment:
Could we make region a command line parameter and default it to US-EAST-1?

Reply:
Agreed. @gopikrishnakolla We should parameterize the region value and any other hardcoded values. Refer to Python samples.